docs/Documentation/Development/Jupyter/index.md

```diff
@@ -46,122 +52,171 @@ axes[3].set_title("fill_between");
+
+The single-user server/web interface. Use to create, save, or load .ipynb notebook files. This is what users generally interact with.
 
 ### **Jupyter Notebook**
-    * An individual .pynb file
-    * Contains your Python code and visualizations
-    * Sharable/downloadable
+
+A Notebook is an individual .pynb file. It contains your Python code and visualizations, and is sharable/downloadable.
 
 ###  **Jupyter lab**
-    * A "nicer" web interface for Jupyter - "notebooks 2.0"
-    * Preferred by some
-    * Lacking some features of "classic" notebooks
+
+A "nicer" redesigned web interface for your Jupyter Server - "Notebooks 2.0". Preferred by some, and promoted as the next evolution of Notebooks.
+Lab has many new and different extensions, but many are also not compatible between Notebook and Lab. Lab is still under development, so is lacking some features of "classic" notebooks.
 
 ### **Kernel**
-    * The Python environment used by a notebook
-    * More on kernels later
 
-## Using Europa
+Kernels define the Python environments used by your notebooks. Derived from ipykernel, a predecessor project to Jupyter: you may see Jupyter kernels referred to as "ipykernels". Custom kernels require the "ipykernel" package installed in your Jupyter conda environment.
+
+More on kernels later.
 
-We run a Jupyterhub server that is available. 
+## Eagle's "Europa" Jupyterhub Server
+
+The NREL HPC team runs a Jupyterhub server called Europa that is available for internal (NREL) Eagle users only. 
+
+Europa is connected to Eagle's Lustre storage system for access to /projects data.
+
+A replacement for Europa on Kestrel is in the planning stage.
 
 ### Europa's Advantages:
-    * Fast and easy access
-    * Use regular Eagle credentials
-    * Great for light to moderate processing/debugging/testing
 
-### Europa's Disadvantages:
-    * Limited resource: 8 cores/128GB RAM per user beefore automatic throttling
-    * Compete with other users for CPU/RAM on a single machine
-    * No custom environments (for now)
+* Fast and easy access to notebooks with no setup.
+* Use regular Eagle credentials to log in.
+* Great for simple tasks, including light to moderate data processing, code debugging/testing, and light to moderate visualization using standard/basic scientific and visualization libraries.
 
+### Europa's Disadvantages:
 
-### Simple Instructions:
-    
-    - Visit Europa at (https://europa.hpc.nrel.gov/)
+* Limited resources: Only 48 CPU cores and 190GB RAM total.
+* Managed usage: Up to 8 cores/128GB RAM per user before automatic throttling will greatly slow down processing.
+* Must compete with other users for CPU and RAM on a single machine.
+* Limited list of scientific libraries and visualization tools are available, and may not be latest versions.
+* Custom environments are difficult to configure.
+* No access for external (non-NREL) users.
+* Not available for Kestrel (yet).
+   
+### Simple Instructions to access Europa:
```

Comment:
Should we remove this other page entirely: https://nrel.github.io/HPC/Documentation/Development/Jupyter/jupyterhub/

I think everything from that page has been covered here

Reply:
Agreed, I think that page could be removed. Should that be part of this PR, or separate?